### PR TITLE
Add pandarallel to ecosystem documentation

### PR DESCRIPTION
**Pull request by nalepae**
As proposed by datapythonista here: https://github.com/nalepae/pandarallel/issues/28

### CODE REVIEW

File: doc/source/ecosystem.rst [hunk 1]
```diff
@@ -327,6 +327,26 @@ PyTables, h5py, and pymongo to move data between non pandas formats. Its graph
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pandaral·lel provides a simple way to parallelize your pandas operations on all your CPUs by changing only one line of code.
+If also displays progress bars.
+
+.. code:: python
+
+    # import pandarallel
+    from pandarallel import pandarallel
+
```
Comment (jbrockmendel) [inline doc/source/ecosystem.rst]: linter is complaining about trailing whitespace

File: doc/source/ecosystem.rst [hunk 2]
```diff
@@ -327,6 +327,26 @@ PyTables, h5py, and pymongo to move data between non pandas formats. Its graph
...
 
+`Pandaral·lel <https://github.com/nalepae/pandarallel>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pandaral·lel provides a simple way to parallelize your pandas operations on all your CPUs by changing only one line of code.
+If also displays progress bars.
+
+.. code:: python
+
+    # import pandarallel
```
Comment (jbrockmendel) [inline doc/source/ecosystem.rst]: these comments arent doing much

File: doc/source/ecosystem.rst [hunk 3]
```diff
@@ -327,6 +327,26 @@ PyTables, h5py, and pymongo to move data between non pandas formats. Its graph
 based approach is also extensible by end users for custom formats that may be
 too specific for the core of odo.
 
+`Pandaral·lel <https://github.com/nalepae/pandarallel>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pandaral·lel provides a simple way to parallelize your pandas operations on all your CPUs by changing only one line of code.
```
Comment (jbrockmendel) [inline doc/source/ecosystem.rst]: Pandaral·lel typo?

File: doc/source/ecosystem.rst (hunk 2 above)
Comment (nalepae) [reply doc/source/ecosystem.rst]: I removed the 2 firsts comments and kept the two last ones.
Please tell me if I also should remove last ones (but I think they are useful).

File: doc/source/ecosystem.rst (hunk 3 above)
Comment (nalepae) [reply doc/source/ecosystem.rst]: Actually not, the point is inspired from [Avinguda del Paral·lel](https://en.wikipedia.org/wiki/Avinguda_del_Paral%C2%B7lel) in Barcelona, Spain.

This point is also present on the [GitHub page of the project](https://github.com/nalepae/pandarallel).

I removed it anyway.

**Comment by alimcmaster1** [comment]
@nalepae is this this active? Mind merging master the CI issues should now be fixed.

**Comment by nalepae** [comment]
@alimcmaster1: Yes this PR (and `pandarallel` itself) are both active.

I'll be happy to be part of the `pandas` ecosystem!

File: doc/source/ecosystem.rst [hunk 4]
```diff
@@ -327,6 +327,21 @@ PyTables, h5py, and pymongo to move data between non pandas formats. Its graph
 based approach is also extensible by end users for custom formats that may be
 too specific for the core of odo.
 
+`Pandarallel <https://github.com/nalepae/pandarallel>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (jreback) [inline doc/source/ecosystem.rst]: i think this needs to be the same length as the title

Comment (nalepae) [reply doc/source/ecosystem.rst]: OK thank you I just fixed it.

**Comment by WillAyd** [comment]
Thanks @nalepae

**Pull request merged**